docs/formatters.md

```diff
@@ -1,143 +1,153 @@
-# A list of known formatters
+# Known formatters
```

Comment:
This file's content is a duplicate of the wiki. Either move the content there, or I'll close the wiki.